src/routes/channels/components/ChannelRow.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+  import { currencySymbols } from '$lib/constants.js'
+  import { convertValue } from '$lib/conversion.js'
+  import { translate } from '$lib/i18n/translations.js'
+  import { settings$ } from '$lib/streams.js'
+  import { BitcoinDenomination, type Channel } from '$lib/types.js'
+  import { formatValueForDisplay, truncateValue } from '$lib/utils.js'
+  import Big from 'big.js'
+  import { channelStatusTolabel } from '../utils.js'
+
+  export let channel: Channel
+
+  const { peerAlias, peerId, balanceRemote, balanceLocal, status } = channel
+  const balanceTotal = Big(balanceLocal).plus(balanceRemote)
+  const localPercent = Big(balanceLocal).div(balanceTotal).times(100).toNumber()
+  const remotePercent = Big(balanceRemote).div(balanceTotal).times(100).toNumber()
```

Comment:
At the moment there is no way for the channel to be updated where this component is rendered, but I updated to be reactive just in case we use that component somewhere else or in future the channel is updated on that route.